vmdb/app/views/ops/_schedule_show.html.haml

```diff
@@ -0,0 +1,55 @@
+#main_div
+  = render(:partial=>"layouts/flash_msg")
+  %p.legend Schedule Info
+  %table.style1
+    %tr
+      %td.key Description
+      %td= h(@selected_schedule.description)
+    %tr
+      %td.key Active
+      %td= h(@selected_schedule.enabled)
+    %tr
+      %td.key Action
+      %td
+        - if @selected_schedule.sched_action[:method] && @selected_schedule.sched_action[:method] == "check_compliance"
+          = h(ui_lookup(:model=>@selected_schedule.towhat))
+          Compliance Check
+        - elsif @selected_schedule.sched_action[:method] && @selected_schedule.sched_action[:method] == "db_backup"
+          Database Backup
+        - else
+          = h(ui_lookup(:model=>@selected_schedule.towhat))
+          Analysis
+    - if @selected_schedule.sched_action[:method] != "db_backup"
+      %tr
+        %td.key Filter
+        - unless @selected_schedule.miq_search
+          - if @selected_schedule.filter.is_a?(MiqExpression)
+            %td
+              - @exp_table.each do |token|
+                - if ! ["AND", "OR", "(", ")"].include?([token].flatten.first)
```

Comment:
oh, I have seen this in some @romanblanco PR

what's that? `[token].flatten.first`?

can we extract that into a method if it's in more places?